worker/worker.go

```diff
@@ -345,6 +347,14 @@ func setEventFailStatus(event *api.Event, ErrorMessage string) {
 	logdog.Error("Operation failed", logdog.Fields{"event": event})
 }
 
+// formatVersionName replace the random name with default name '$createTime|$commitID' when name empty in create version
+func formatVersionName(event *api.Event) {
```

Comment:
This is a very easy function, does not need to take the complex `*api.Event` as the param, just needs the commit info. What about `getVersionName(commit string) string`? This function will return the version name.